.github/workflows/build.yml

```diff
@@ -15,7 +15,9 @@ on:
 jobs:
   test:
     runs-on: ${{ matrix.os }}
-
+    defaults:
+      run:
+        working-directory: "."
```

Comment:
Is it necessary to specify the current directory is the default one?

Reply:
Yes because of the sub directory. Otherwise the workflows fail.